packages/slice-machine/server/src/api/index.ts

```diff
@@ -238,16 +219,30 @@ router.post(
 
 router.post(
   "/auth/status",
-  async function (
-    _req: express.Request,
+  WithEnv(async function (
+    req: RequestWithEnv,
     res: express.Response
   ): Promise<Express.Response> {
-    const payload = await statusAuth();
+    const payload = await statusAuth(req);
     if (payload.status === "error") {
       return res.status(500).json(payload);
     }
     return res.status(200).json(payload);
-  }
+  })
+);
+
+router.get(
```

Comment:
this one is doubled, there is already a state route in this file.

Reply:
Oh true, I may have copy paste for some reason and forgot to delete it. Nice catch